edweiss-app/app/(app)/(tabs)/index.tsx

```diff
@@ -1,113 +1,20 @@
 
-import For from '@/components/core/For';
 import TText from '@/components/core/TText';
-import TScrollView from '@/components/core/containers/TScrollView';
-import TTouchableOpacity from '@/components/core/containers/TTouchableOpacity';
-import TView from '@/components/core/containers/TView';
 import RouteHeader from '@/components/core/header/RouteHeader';
-import FancyButton from '@/components/input/FancyButton';
-import FancyTextInput from '@/components/input/FancyTextInput';
-import { Collections, callFunction } from '@/config/firebase';
-import t from '@/config/i18config';
-import ReactComponent, { ApplicationRoute } from '@/constants/Component';
-import { useDynamicDocs } from '@/hooks/firebase/firestore';
-import Memento from '@/model/memento';
-import { router } from 'expo-router';
-import React, { useState } from 'react';
+import { ApplicationRoute } from '@/constants/Component';
+import React from 'react';
 
 const HomeTab: ApplicationRoute = () => {
```

Comment:
I just had a quick question. As you deleted a lot of code from  the index.tsx, if I remember correctly, it because it was ment to be the HomePage that @Gwen678 has been working on ?

Reply:
Yup